--- MuseScore-4.0.2/build/Linux+BSD/org.musescore.MuseScore.desktop.in~	2023-03-13 12:59:42.000000000 +0000
+++ MuseScore-4.0.2/build/Linux+BSD/org.musescore.MuseScore.desktop.in	2023-03-16 22:21:45.069756860 +0000
@@ -8,9 +8,11 @@
 GenericName=Music notation
 GenericName[de]=Notensatz
 GenericName[fr]=Notation musicale
+GenericName[pl]=Zapis nutowy
 Comment=Create, play and print beautiful sheet music
 Comment[ru]=Визуальный редактор нотных партитур
 Comment[fr]=Gravure de partitions musicales
+Comment[pl]=Twórz, odtwarzaj i drukuj partytury
 Icon=mscore@MSCORE_INSTALL_SUFFIX@
 Exec=mscore@MSCORE_INSTALL_SUFFIX@ %F
 Terminal=false
