--- MuseScore-4.1.1/build/Linux+BSD/org.musescore.MuseScore.desktop.in~	2023-07-25 18:38:26.000000000 +0000
+++ MuseScore-4.1.1/build/Linux+BSD/org.musescore.MuseScore.desktop.in	2023-08-06 09:02:12.915705506 +0000
@@ -8,9 +8,11 @@
 GenericName=Music notation
 GenericName[de]=Notensatz
 GenericName[fr]=Notation musicale
+GenericName[pl]=Zapis nutowy
 Comment=Create, play and print beautiful sheet music
 Comment[ru]=Визуальный редактор нотных партитур
 Comment[fr]=Gravure de partitions musicales
+Comment[pl]=Twórz, odtwarzaj i drukuj partytury
 Icon=mscore@MUSESCORE_INSTALL_SUFFIX@
 Exec=mscore@MUSESCORE_INSTALL_SUFFIX@ %F
 Terminal=false
