--- MuseScore-3.6.2/main/CMakeLists.txt~	2021-02-08 12:16:37.000000000 +0000
+++ MuseScore-3.6.2/main/CMakeLists.txt	2021-03-03 21:33:31.167705900 +0000
@@ -226,22 +226,6 @@
 else (MINGW)
 
    if ( NOT MSVC )
-      ## install qwebengine core
-      if (NOT APPLE AND USE_WEBENGINE)
-         install(PROGRAMS
-            ${QT_INSTALL_LIBEXECS}/QtWebEngineProcess
-            DESTINATION bin
-            )
-         install(DIRECTORY
-            ${QT_INSTALL_DATA}/resources
-            DESTINATION lib/qt5
-            )
-         install(DIRECTORY
-            ${QT_INSTALL_TRANSLATIONS}/qtwebengine_locales
-            DESTINATION lib/qt5/translations
-            )
-      endif(NOT APPLE AND USE_WEBENGINE)
-
       set_target_properties (
          mscore
          PROPERTIES
