diff -dur MuseScore-3.0.4.orig/mscore/CMakeLists.txt MuseScore-3.0.4/mscore/CMakeLists.txt
--- MuseScore-3.0.4.orig/mscore/CMakeLists.txt	2019-02-28 12:31:10.000000000 +0100
+++ MuseScore-3.0.4/mscore/CMakeLists.txt	2019-03-10 16:43:03.375054036 +0100
@@ -699,21 +699,6 @@
 else (MINGW)
 
    if ( NOT MSVC )
-## install qwebengine core
-      if (NOT APPLE AND USE_WEBENGINE)
-         install(FILES
-            ${QT_INSTALL_LIBEXECS}/QtWebEngineProcess
-            DESTINATION bin
-            )
-         install(DIRECTORY
-            ${QT_INSTALL_DATA}/resources
-            DESTINATION lib/qt5
-            )
-         install(DIRECTORY
-            ${QT_INSTALL_TRANSLATIONS}/qtwebengine_locales
-            DESTINATION lib/qt5/translations
-            )
-      endif(NOT APPLE AND USE_WEBENGINE)
 
       target_link_libraries(mscore
          ${ALSA_LIB}
