--- MuseScore-4.1.1/src/diagnostics/CMakeLists.txt~	2023-07-25 18:38:26.000000000 +0000
+++ MuseScore-4.1.1/src/diagnostics/CMakeLists.txt	2023-08-06 08:59:41.430792298 +0000
@@ -134,7 +134,7 @@
             endif()
         endif()
 
-        install(PROGRAMS ${MUE_CRASHPAD_HANDLER_PATH} DESTINATION bin)
+	#        install(PROGRAMS ${MUE_CRASHPAD_HANDLER_PATH} DESTINATION bin)
     elseif(OS_IS_WIN)
         if (NOT MUE_CRASHPAD_HANDLER_PATH)
             set(MUE_CRASHPAD_HANDLER_PATH ${CPAD_ROOT_PATH}/windows/x86-64/crashpad_handler.exe)
