--- MuseScore-4.1.1/CMakeLists.txt~	2023-07-25 18:38:26.000000000 +0000
+++ MuseScore-4.1.1/CMakeLists.txt	2023-08-06 09:05:22.472378347 +0000
@@ -46,7 +46,7 @@
 # - vtest           - for visual tests (for CI)
 # - utest           - for unit tests (for CI)
 
-set(MUSESCORE_BUILD_MODE "dev" CACHE STRING "Build mode")
+set(MUSESCORE_BUILD_MODE "release" CACHE STRING "Build mode")
 # Possible MUSESCORE_BUILD_MODE values:
 # - dev     - for development/nightly builds
 # - testing - for testing versions (alpha, beta, RC)
