--- MuseScore-4.0.2/CMakeLists.txt~	2023-03-13 12:59:42.000000000 +0000
+++ MuseScore-4.0.2/CMakeLists.txt	2023-03-16 22:48:04.570633760 +0000
@@ -40,7 +40,7 @@
 ###########################################
 
 set(MUSESCORE_REVISION "" CACHE STRING "Build revision")
-set(MUSESCORE_BUILD_CONFIG "dev" CACHE STRING "Build config")
+set(MUSESCORE_BUILD_CONFIG "release" CACHE STRING "Build config")
 # Possible MUSESCORE_BUILD_CONFIG values:
 # - dev     - for development/nightly builds
 # - testing - for testing versions (alpha, beta, RC)
