diff -dur -x '*~' MuseScore-3.1.0.orig/share/sound/CMakeLists.txt MuseScore-3.1.0/share/sound/CMakeLists.txt
--- MuseScore-3.1.0.orig/share/sound/CMakeLists.txt	2019-05-28 09:33:38.000000000 +0200
+++ MuseScore-3.1.0/share/sound/CMakeLists.txt	2019-06-04 09:44:21.000000000 +0200
@@ -29,13 +29,13 @@
       )
 endif ()
 
-if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/MuseScore_General-License.md)
+if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/MuseScore_General_License.md)
   install (FILES FluidR3Mono_License.md
       DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}sound
-      RENAME MuseScore_General-License.md)
+      RENAME MuseScore_General_License.md)
 else ()
   install(FILES
-      MuseScore_General-License.md
+      MuseScore_General_License.md
       DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}sound
       )
 endif ()
