--- MuseScore-3.6.2/build/FindQt5.cmake~	2021-02-08 12:16:37.000000000 +0000
+++ MuseScore-3.6.2/build/FindQt5.cmake	2021-03-03 22:35:06.449357445 +0000
@@ -47,7 +47,7 @@
 
 include_directories(${QT_INCLUDES})
 
-find_program(QT_QMAKE_EXECUTABLE qmake)
+find_program(QT_QMAKE_EXECUTABLE qmake-qt5)
 set(_qmake_vars
     QT_INSTALL_ARCHDATA
     QT_INSTALL_BINS
--- MuseScore-3.6.2/CMakeLists.txt~	2021-02-08 12:16:37.000000000 +0000
+++ MuseScore-3.6.2/CMakeLists.txt	2021-03-03 22:36:02.887616621 +0000
@@ -25,7 +25,7 @@
 include(build/functions.cmake) # library of CMake functions ("fn__" namespace)
 
 # Print Qt version or fail the build if Qt (qmake) is not in PATH.
-fn__require_program(QMAKE Qt --version "https://musescore.org/en/handbook/developers-handbook/compilation" qmake)
+fn__require_program(QMAKE Qt --version "https://musescore.org/en/handbook/developers-handbook/compilation" qmake-qt5)
 
 # Libraries linked via full path no longer produce linker search paths.
 cmake_policy(SET CMP0003 NEW)
